dotnet/src/Experimental/Assistants/AssistantBuilder.cs

```diff
@@ -155,7 +155,10 @@ public AssistantBuilder WithPlugin(ISKPlugin plugin)
     /// <returns><see cref="AssistantBuilder"/> instance for fluid expression.</returns>
     public AssistantBuilder WithPlugins(IEnumerable<ISKPlugin> plugins)
     {
-        this._plugins.AddRange(plugins);
+        foreach (ISKPlugin plugin in plugins)
+        {
+            this._plugins.Add(plugin);
+        }
```

Comment:
Nit: Can we not use `SKPluginCollection.AddRange` here?

Reply:
> Can we not use SKPluginCollection.AddRange here?

We can. I'd replaced this when I removed the extension method and neglected to add it back when I put the method onto the concrete type.